### PR TITLE
Add eth light client update test

### PR DESCRIPTION
**Pull request by yjcbert**

### CODE REVIEW

File: modules/tibc/light-clients/09-eth/types/update_test.go [hunk 1]
```diff
@@ -13,18 +13,81 @@ import (
...
+	genesisHeight := height - 200
+
+	header, err := GetNodeHeader(rc, ethurl, genesisHeight)
+	suite.NoError(err)
+
+	number := clienttypes.NewHeight(0, header.Number.Uint64())
+
+	clientState := exported.ClientState(&tibcethtypes.ClientState{
+		Header:          header.ToHeader(),
+		ChainId:         56,
```
Comment (chengwenxi) [inline modules/tibc/light-clients/09-eth/types/update_test.go:44]: chainId == 56?

Comment (chengwenxi) [reply modules/tibc/light-clients/09-eth/types/update_test.go:44]: I don't know why the test works. Do we not check the chainId?

Comment (2841156671) [reply modules/tibc/light-clients/09-eth/types/update_test.go:44]: forget change 56 to 1

File: modules/tibc/light-clients/09-eth/types/update_test.go [hunk 2]
```diff
@@ -60,14 +59,16 @@ func (suite *ETHTestSuite) TestCheckHeaderAndUpdateState() {
...
+	for i := uint64(1); i <= uint64(1.5*float64(epoch)); i++ {
+
 		updateHeader, err := GetNodeHeader(rc, ethurl, genesisHeight+i)
 
 		// skip some connection error on getting header
 		if err != nil {
 			i--
 			continue
 		}
+		fmt.Println("test :", i)
```
Comment (chengwenxi) [inline modules/tibc/light-clients/09-eth/types/update_test.go:71]: delete

Comment (2841156671) [reply modules/tibc/light-clients/09-eth/types/update_test.go]: OK

**Review by chengwenxi — APPROVED** [review]

**Review by aofengli — APPROVED** [review]

**Pull request merged**